webui/src/app/--/bundles/[bundle]/reload+api.ts

```diff
@@ -14,7 +14,11 @@ export async function GET(_request: Request, params: Record<'bundle', string>) {
       );
     }
 
-    return Response.redirect(bundle.serializeOptions.sourceUrl, 302);
+    // Convert the source URL to localhost, avoiding "unauthorized requests" in Metro
+    const sourceUrl = new URL(bundle.serializeOptions.sourceUrl);
+    sourceUrl.hostname = 'localhost';
```

Comment:
nit: we could add more host to allowed host if needed: https://github.com/expo/expo/blob/102bb32d4ddad5217733e84b7d995634c72aee48/packages/%40expo/cli/src/start/server/middleware/CorsMiddleware.ts#L5-L19

Reply:
Yeah, but likely not the "current LAN address", right? Basically, when opening Atlas through the <kbd>shift</kbd>+<kbd>m</kbd> menu, it opens it on the LAN address.

When Atlas detects a bundle change, without the bundle actually being rebuilt, we need to trigger that manually (just through a fetch). I don't see a good way to do that through LAN addresses with this.

Let's just keep this for now.